### ISSUE TITLE
CommandSource.sendMessage is uncallable.

### ISSUE THREAD

**Issue by unnecessarymb** [CraftTweaker/CraftTweaker#1679]
### Issue description

Due to requiring the non-existant type UUID CommandSource.sendMessage is uncallable.

### Steps to reproduce

Stare at the code and documentation unable to find a way to call CommandSource.sendMessage.

### Script used

N/A

### The crafttweaker.log file

N/A

### Minecraft version

1.18

### Modloader

Forge

### Modloader version

40.2.10

### CraftTweaker version

1.18.2-9.1.209

### The latest.log file

N/A

**Comment by unnecessarymb** [comment]
Even if there is an undocumented typecast, there is no way to find the UUID of many CommandSources.

**Comment by jaredlll08** [comment]
This will be resolved with a language update that will happen hopefully soon, but no promises.

**Comment by jaredlll08** [comment]
This has been fixed by c90f2dee03e16cfaa99a591a62f63691ad8907d3

```
import uuid.UUID;

UUID.parse("019fe22d-aa21-4919-bd30-a3c9e4ef80d2");
```
